mongodb_atlas/manifest.json

```diff
@@ -68,13 +68,14 @@
     },
     "dashboards": {
       "MongoDB-Atlas-Overview": "assets/dashboards/MongoDB-Atlas-Overview_dashboard.json",
-      "MongoDB-Atlas-Vector-Search-Overview": "assets/dashboards/MongoDB-Atlas-Vector-Search-Overview_dashboard.json"
+      "MongoDB-Atlas-Vector-Search-Overview": "assets/dashboards/MongoDB-Atlas-Vector-Search-Overview_dashboard.json",
+      "MongoDB-dbStats-collStats-Dashboard": "assets/dashboards/MongoDB-Atlas-dbStats-collStats_dashboard.json"
     }
   },
   "author": {
-    "support_email": "help@datadoghq.com",
-    "name": "Datadog",
-    "homepage": "https://www.datadoghq.com",
-    "sales_email": "info@datadoghq.com"
+    "support_email": "frank.sun@mongodb.com",
```

Comment:
Is there a generic support email in case frank ever leaves?

Reply:
We are in the process of creating it. We'll update this field once we have it.